### PR TITLE
feat(locksmith) - rename address to erc20address

### PR DESCRIPTION
**Pull request by kalidiagne**
# Description

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: packages/unlock-js/openapi.yml [hunk 1]
```diff
@@ -1324,7 +1324,7 @@ paths:
             type: number
 
         - in: query
-          name: address
+          name: erc20Address
```
Comment (searchableguy) [inline packages/unlock-js/openapi.yml:1327]: I would not change these. Just the internal naming, that's all.

Comment (kalidiagne) [reply packages/unlock-js/openapi.yml:1327]: It will not be a breaking change here we use it with the `operationId` in `unlock-app`

Comment (searchableguy) [reply packages/unlock-js/openapi.yml:1327]: It is still a breaking change externally outside of our storage service and I don't see good reason to do that so I will keep it as it is but internally refactor.

Comment (kalidiagne) [reply packages/unlock-js/openapi.yml]: restored

**Pull request merged**